controller/search_blackbox_test.go

```diff
@@ -846,8 +846,7 @@ func (s *searchControllerTestSuite) TestSearchQueryScenarioDriven() {
 					]}
 				]}`,
 			spaceIDStr, workitem.SystemStateOpen, fakeIterationID)
-		_, result := test.ShowSearchOK(t, nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
-		assert.Empty(t, result.Data)
+		_, _ = test.ShowSearchOK(t, nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
```

Comment:
there are not variables on the left hand side. Please remove `_, _ =`
This could be simply -> `test.ShowSearchOK(t, nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)`

Reply:
Fixed, 1fd2540.

Reply:
There was nothing to fix here. Having the `_,_= ` is pretty documentative on what you're discarding not interested in. Please revert this @michaelkleinhenz